### ISSUE TITLE
Update run configurations when a feature file is moved with Eclipse

### ISSUE THREAD

**Issue by Flaburgan** [cucumber/cucumber-eclipse#238]
**Steps to reproduce:**
Create a feature file
Move it to another folder
Run the feature file by right clicking it it, Run as, Cucumber Feature

**What happens:**
Exception in thread "main" java.lang.IllegalArgumentException: Not a file or directory: [the old path of the file]

**What should happen:**
The run configuration should be updated to use the new path of the file, and the file should be run properly.

**Comment by futrethegreat** [comment]
> Steps to reproduce:
> Create a feature file
> Move it to another folder
> Run the feature file by right clicking it it, Run as, Cucumber Feature
> 
> What happens:
> Exception in thread "main" java.lang.IllegalArgumentException: Not a file or directory: [the old path of the file]
> 
> What should happen:
> The run configuration should be updated to use the new path of the file, and the file should be run properly.

Suffered this as well...

**Comment by Flaburgan** [comment]
You can fix that manually by clicking right -> run as -> run configuration and update the feature path.

**Comment by futrethegreat** [comment]
> You can fix that manually by clicking right -> run as -> run configuration and update the feature path.
Yes, I found that work arround but it will be more profesional if the IDE could do it automatically when any file is moved. Thank you for your advise! 👍

**Comment by girijant** [comment]
@Flaburgan, @futrethegreat,
Already aware of this issue and this is the work around. Let me analyse first and if possible we have to fix it.

Thanks,
Girija

**Comment by futrethegreat** [comment]
Thank you very much!
